.github/workflows/release.yml

```diff
@@ -0,0 +1,80 @@
+name: Release packages
+
+on:
+  push:
+    tags:
+      - '*'
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  basepkgs:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: Vanilla amd64
+            board: amd64
+            zmq: "false"
+            parallel: "$(nproc)"
+            image: basefocal:20.04.0
+          - name: ZMQ amd64
+            board: amd64
+            zmq: "true"
+            parallel: "$(nproc)"
+            image: basefocal:20.04.0
+          - name: Vanilla aarch64
+            board: aarch64
+            zmq: "false"
+            parallel: "$(nproc)"
+            image: basefocal:20.04.0
+          - name: ZMQ aarch64
+            board: aarch64
+            zmq: "true"
+            parallel: "$(nproc)"
+            image: basefocal:20.04.0
+          - name: Vanilla rpi4
+            board: rpi4
+            zmq: "false"
+            parallel: 2
+            image: basefocal:20.04.0
+          - name: ZMQ rpi4
+            board: rpi4
+            zmq: "true"
+            parallel: 2
+            image: basefocal:20.04.0
+          - name: Vanilla odroid-xu4
+            board: odroid-xu4
+            zmq: "false"
+            parallel: 2
+            image: qnnpack:20.04.0
+          - name: ZMQ odroid-xu4
+            board: odroid-xu4
+            zmq: "true"
+            parallel: 2
+            image: qnnpack:20.04.0
+          - name: Vanilla lx2160
+            board: lx2160
+            zmq: "false"
+            parallel: "$(nproc)"
+            image: basefocal:20.04.0
+          - name: ZMQ lx2160
+            board: lx2160
+            zmq: "true"
+            parallel: "$(nproc)"
+            image: basefocal:20.04.0
+
+    name: ${{ matrix.name }}
+    uses: klepsydra-technologies/kpe-ghworkflows/.github/workflows/generate_release.yml@v2.0.6
+    with:
+      board: ${{ matrix.board }}
+      image: ${{ matrix.image }}
+      parallel: ${{ matrix.parallel }}
+      packages: DEB,TGZ
+      testonly: false
+      cmakeopts: >
+        -DKPSR_WITH_DOXYGEN=true -DKPSR_ENABLE_EXAMPLES=false -DKPSR_WITH_SOCKET=true
+        -DCMAKE_BUILD_TYPE=Release -DKPSR_WITH_ZMQ=${{ matrix.zmq }}
```

Comment:
there is no different version for socket and zmq. So I think you can remove them from the matrix.